source/_posts/2023-10-14-home-assistant-os-release-11.markdown

```diff
@@ -0,0 +1,56 @@
+
+Enjoy the latest version of Home Assistant OS!
+
+Stefan
+
+<!--more-->
+
+- [Linux' preemptible kernel configuration](#linux-preemptible-kernel-configuration)
+- [VM filesystem freeze is being relayed to Home Assistant](#vm-filesystem-freeze-is-being-relayed-to-home-assistant)
+- [Docker and containerd Upgrades](#docker-and-containerd-upgrades)
+- [More Highlights in Home Assistant OS 11](#more-highlights-in-home-assistant-os-11)
+
+
+## Linux' preemptible kernel configuration
+We've applied Linux's preemptible kernel configuration across all boards. This results in lower latencies even on busy systems (e.g. due to slow I/O operations), making your smart home even more responsive.
+
+## VM filesystem freeze is being relayed to Home Assistant
+This is a neat feature for more advanced setups based on Proxmox (or other KVM based VMs).
+Home Assistant's recorder integration uses a database underneath (by default SQLite). When  Home Assistant takes a backup, the Supervisor notifies the database engine  before copying the database files (currently this is implemented for SQLite and MariaDB). With that notification the database engine can take the necessary steps to ensure that the database files are in a consistent state before the backup is taking place.
+However, when creating a snapshot using the VM snapshot feature the database doesn’t know about this, hence the snapshot can end up with an inconsistent state of the database. On snapshot restore, the database may or may not be able to recover from that inconsistent state, which can lead to partial or even complete data loss of the recorder data.
+With Home Assistant OS 11, on Proxmox/KVM-based VMs, when using the snapshot feature, the file system freeze is now relayed to Home Assistant. Home Assistant then uses the same notification mechanism as backups are using. This ensures that VM snapshots are always coherent, making sure rollbacks of your smart home systems are reliable.
+
+## Docker and containerd Upgrades
+
+In this release, Home Assistant OS has adopted the latest versions of Docker (v24.0.6) and containerd (v1.7.6), ensuring better performance and container management. We’ve also improved the containerd configuration to drop unnecessary components. With this containerd uses less CPU and memory resources, ensuring better overall performance.
+
+## More Highlights in Home Assistant OS 11
+
+- **Consistent network interface naming**: On Arm-based boards, network names are now enumerated based on the device tree. This means that the first Ethernet device will no longer be named eth0 but end0. The same network configuration used previously is automatically applied to the network interface with the new name.
+**This can be a breaking change ⚠️**: If you use the name of the Ethernet interface in custom scripts or automations you'll have to adjust to the new name (as shown in the network settings)!
```

Comment:
```suggestion
**This can be a breaking change ⚠️**: If you use the name of the Ethernet interface in custom scripts or automation, you'll have to adjust to the new name (as shown in the network settings)!
```